### DISCUSSION TITLE
Is it possible to turn the pool pump on via relay

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#731] [General]
I have a Nixie pool controller which has a Pentair VS pump which is connected to the Sequent 8 via 4 relays. Each relay controls a specific speed programmed on the pump. This thing is awesome, I have taken my completely dumb system to one I can monitor and control from practically anywhere -> Thanks!

Even though I live in Florida, my pool is inside a roof covered enclosure (no sun directly hits the pool). My pool is at 71F this morning while my neighbors is at a nice 82F. Since the air conditioning runs most of the day I have decided to install a heat exchanger (HX) ([https://www.hotspotenergy.com/pool-heater/](url)) which bypasses the A/C condensing unit and uses the waste heat to heat the pool.

Background: I already have the circuit 'ok' relay of my two DC power supplies hooked into REM its output sent to the pool controller (via two aux circuits, one for the 5VDC and one for the 12VDC). I can then see it in my home assistant. There is probably a more elegant way to do this but I haven't explored much more until I get the whole system up and running. 

When the heat exchanger is active I need the main filter pump to be on. My thought right now is to integrate that into the Nixie somehow, similar to what I have done with the power supply ok signals. I have a 24VAC power relay which I can take the 24VAC signal from the HX controller and route through the power relay to act as a dry contact signal to REM (via Mega-BAS). I'm not sure how I could get the pool controller to turn on my pump based on the REM relay contact closing. Is setting up an Aux Circuit and then Circuit Group the best way?

**Comment by rstrouse** [comment]
You would create a feature for the HX which would assign a speed that the pump should run at.  Then you can do a couple of things to get the signal converted into something meaningful.  The easiest method would be to use a relay with a 24vac coil, then use this relay to trigger the dry contact.  This will ensure the pump is running at the minimum speed whenever the HX is energized while isolating the two systems electrically.

**Reply by subcritical71** [reply]
Great, thanks. I will give that a try. The relay I have is like you specified.
